zerver/views/video_calls.py

```diff
@@ -151,3 +153,35 @@ def deauthorize_zoom_user(request: HttpRequest) -> HttpResponse:
             auth=(settings.VIDEO_ZOOM_CLIENT_ID, settings.VIDEO_ZOOM_CLIENT_SECRET),
         ).raise_for_status()
     return json_success()
+
+
+@zulip_login_required
+@never_cache
+def get_bbb_url(request: HttpRequest, user_profile: UserProfile) -> HttpResponse:
+    id = "zulip-" + str(random.randint(100000000000, 999999999999))
+    password = ''.join(random.SystemRandom().choice(string.ascii_uppercase + string.digits) for _ in range(10))
```

Comment:
Can you add a link to the BigBlueButton documentation on what characters are allowed for id / password in a comment here?

Reply:
(And maybe also on the checksum parameter)?